xml/System.Threading/Timer.xml

```diff
@@ -716,8 +716,8 @@ Sub TimerCallback(state As Object)
         <Parameter Name="period" Type="System.TimeSpan" />
       </Parameters>
       <Docs>
-        <param name="dueTime">A <see cref="T:System.TimeSpan" /> representing the amount of time to delay before invoking the callback method specified when the <see cref="T:System.Threading.Timer" /> was constructed. Specify negative one (-1) milliseconds to prevent the timer from restarting. Specify zero (0) to restart the timer immediately.</param>
-        <param name="period">The time interval between invocations of the callback method specified when the <see cref="T:System.Threading.Timer" /> was constructed. Specify negative one (-1) milliseconds to disable periodic signaling.</param>
+        <param name="dueTime">A <see cref="T:System.TimeSpan" /> representing the amount of time to delay before invoking the callback method specified when the <see cref="T:System.Threading.Timer" /> was constructed. Specify <see cref="F:System.Threading.Timeout.Infinite" /> milliseconds to prevent the timer from restarting. Specify zero (0) to restart the timer immediately.</param>
+        <param name="period">The time interval between invocations of the callback method specified when the <see cref="T:System.Threading.Timer" /> was constructed. Specify <see cref="F:System.Threading.Timeout.Infinite" /> to disable periodic signaling.</param>
```

Comment:
These arguments are TimeSpans. If the goal is to change them to use members of Timeout, they should use InfiniteTimeSpan, not Infinite.

Reply:
Somehow a bad replace.. I am sorry I will correct all of these asap.

Reply:
Done.

Reply:
Hopefully thats everything if not I will get it right eventually.